src/pages/Script/styles.module.scss

```diff
@@ -23,30 +23,33 @@
   width: 100%;
```

Comment:
Some changes are introduced by local linter

Reply:
The code has been configured with automatic formatting for preservation. Is the issue here referring to automatic line breaks?

Reply:
Changes in this file cover `auto blank line insertion` and `different wrap rule` at https://github.com/Magickbase/ckb-explorer-frontend/pull/4/files/1fb945c437c81dc490c4f5956ae011193c16dc13#diff-8e04ae6888bd6cfe2f387c1cd287aec98acf334e382f165e6c6f3513d5301946R48-R52

Are they formatted by the linter defined in the project or the local linter, e.g., installed in IDE.

Reply:
There was a conflict with the local CSS formatting, but it has been resolved and unified now.